MC/analysis_testing/o2dpg_analysis_test_workflow.py

```diff
@@ -202,12 +203,16 @@ def get_additional_workflows(input_aod):
                     found_O2zdc_001 = True
                 if "O2bc_001" in j.GetName():
                     found_O2bc_001 = True
+                if "O2trackextra" in j.GetName():
+                    found_O2trackextra_001 = True 
```

Comment:
Is this the same logic as for the other tables?
I see you are checking for `O2trackextra` and the you set `found_O2trackextra_001` to `True`.
In the other cases it was looking for the table with `_001` and then setting the corresponding flag.

Could you please explain? The logic is not quite clear to me.

Reply:
Hi @benedikt-voelkel , indeed I miswrote the table to be checked. If you see in the O2 PR that I linked, we have to check `O2trackextra_001`